build-utils/cmake/modules/FindFbx.cmake

```diff
@@ -61,7 +61,9 @@ function(FindFbxLibrariesGeneric _FBX_ROOT_DIR _OUT_FBX_LIBRARIES _OUT_FBX_LIBRA
   # Figures out matching compiler/os directory.
   
   if("x${CMAKE_CXX_COMPILER_ID}" STREQUAL "xMSVC")
-    if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 19)
+    if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 19.15)
```

Comment:
Visual Studio compiler starts at version 19.10: https://blogs.msdn.microsoft.com/vcblog/2017/11/15/side-by-side-minor-version-msvc-toolsets-in-visual-studio-2017/